<launch>
  <!-- The camera_firewire package and associated parameters -->
  <include file="$(find camera1394v2)/launch/flyatar_camera.launch" />

  <!-- various image parameters -->
  <param name="diff_threshold" type="double" value="65"/>
  <param name="contour_count_max" type="double" value="2"/>
  <param name="image_processor_display_images" type="boolean" value="false"/>

  <!-- Coordinate Systems parameters -->
  <param name="ImageProcessor_OutputCoordinates" type="string" value="Camera"/>

  <include file="$(find plate_tf)/launch/plate_camera_transforms.launch" />

  <include file="$(find flyatar_calibration)/calibration_data/camera_plateimage_calibration_data.launch" />
  <include file="$(find flyatar_calibration)/calibration_data/robotimage_calibration_data.launch" />

  <node pkg="track_image_contours" type="ImageProcessor.py" name="ImageProcessor" />
  <node pkg="track_image_contours" type="ContourIdentifierTF.py" name="ContourIdentifierTF" />
</launch>
